composeApp/src/commonMain/kotlin/sp/bvantur/tasky/core/ui/navigation/TaskyNavHost.kt

```diff
@@ -34,7 +38,24 @@ internal fun TaskyNavHost() {
             navigateToHome = navController::navigateToHome
         )
         registerScreen(onNavigateBack = navController::navigateUp)
-        homeScreen(onCreateEventAction = navController::navigateToCreateEvent)
-        createEvent(onNavigateBack = navController::navigateUp)
+        homeScreen(onCreateEventAction = navController::navigateToCreateEventScreen)
+        createEventScreen(
+            onNavigateBack = navController::navigateUp,
+            onOpenSingleInputScreen = navController::navigateToSingleInputScreen
+        )
+        singleInputScreen(onNavigateBack = navController::navigateUp, onSaveAction = { value, isTitle ->
+            if (isTitle) {
```

Comment:
Personally, when having a field with two options where one is not directly the negated variant of the other, I'd prefer an enum:
```
enum class InputType {
    TITLE, DESCRIPTION
}
```
For a boolean like `isEnabled` it's 100% clear what `!isEnabled` means. For `!isTitle` it's not immediately clear to someone new to the codebase as there might be multiple different field types.